lib/dangermattic/plugins/common/common_release_checker.rb

```diff
@@ -26,6 +26,13 @@ module Danger
   class CommonReleaseChecker < Plugin
     DEFAULT_INTERNAL_RELEASE_NOTES = 'RELEASE-NOTES.txt'
 
+    MESSAGE_STORE_FILE_NOT_CHANGED = 'The `%s` file should be updated if the editorialised release notes file `%s` is being changed.'
```

Comment:
Nitpick: I think we mostly use US English rather than UK English in most of our wording at a8c?
```suggestion
    MESSAGE_STORE_FILE_NOT_CHANGED = 'The `%s` file should be updated if the editorialized release notes file `%s` is being changed.'
```